test/integration/README.md

```diff
@@ -64,6 +64,18 @@ Results at: ./test/integration/render-tests/index.html
 Done in 2.32s.
 ```
 
+
+### Enable ANGLE configuration on render tests
+
+While running render tests locally, you can opt to use the chrome flag `--use-angle` to choose a graphics backend for ANGLE. Some ANGLE backends may not be as performant on certain devices which may affect render tests. See `chrome://flags/#use-angle` for more information on the `--use-angle` flag.
+
+To run render tests with a specfic ANGLE backend:
+```
+USE_ANGLE={INPUT} yarn run test-render
+```
+
+Accepted inputs for `USE_ANGLE` are `metal`, `gl`, `vulkan`, `swiftshader`, and `gles`.
+
```

Comment:
Maybe elaborate on the case where this is helpful as you do in the PR description?
> Some devices (e.g. M1 macs) seem to run the render tests with significantly less failures when forcing the ANGLE backend to be set to OpenGL. To configure the ANGLE backend, you can set the --use-angle input value to USE_ANGLE in CLI like so:
> USE_ANGLE=gl yarn test-render